src/vs/workbench/browser/parts/editor/editorPane.ts

```diff
@@ -163,6 +164,16 @@ export abstract class EditorPane extends Composite implements IEditorPane {
 		this._group = group;
 	}
 
+	getContainingWindow() {
+		let codeWindow;
```

Comment:
Not sure when `group` is undefined, so instead of doing the check and fetching in every place, I added this helper.